### PR TITLE
bugfix: Check is lib and /usr/sbin paths exists before listing

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalacenter/bloop/issues/2403

### CODE REVIEW

File: cli/src/main/scala/bloop/cli/util/OsLibc.scala [hunk 1]
```diff
@@ -40,12 +40,15 @@ object OsLibc {
       if (foundMusl)
         Some(true)
       else {
-        val inLib = os.list(os.Path("/lib")).map(_.last)
+        val libPath = os.Path("/lib")
+        val inLib = if (libPath.toIO.exists()) os.list(libPath).map(_.last) else Nil
```
Comment (adpi2) [inline cli/src/main/scala/bloop/cli/util/OsLibc.scala:44]: A small suggestion:
```suggestion
        val libPath = Some(os.root / "lib").filter(os.exists)
        val inLib = libPath.flatMap(os.list).map(_.last)
```

File: cli/src/main/scala/bloop/cli/util/OsLibc.scala [hunk 2]
```diff
@@ -40,12 +40,15 @@ object OsLibc {
...
         if (inLib.exists(_.contains("-linux-gnu"))) Some(false)
         else if (inLib.exists(name => name.contains("libc.musl-") || name.contains("ld-musl-")))
           Some(true)
         else {
-          val inUsrSbin = os.list(os.Path("/usr/sbin")).map(_.last)
+          val sbinPath = os.Path("/usr/sbin")
+          val inUsrSbin =
+            if (sbinPath.toIO.exists()) os.list(sbinPath).map(_.last) else Nil
```
Comment (adpi2) [inline cli/src/main/scala/bloop/cli/util/OsLibc.scala:51]:
```suggestion
          val sbinPath = Some(os.root / "usr" / "sbin").filter(os.exists)
          val inUsrSbin = sbinPath.flatMap(os.list).map(_.last)
```

**Review by adpi2 — APPROVED** [review]
Some suggestions. But overall it looks good to me.

File: cli/src/main/scala/bloop/cli/util/OsLibc.scala (hunk 1 above)
Comment (tgodzik) [reply cli/src/main/scala/bloop/cli/util/OsLibc.scala]: I modified it to Seq instead of Some because I would have to do `toIterable` otherwise, same below

**Pull request merged**